### PR TITLE
List serviceaccounts

### PR DESCRIPTION
**Pull request by eleijonmarck**
- Add listing all service accounts
- Rearrange UI columns for service account
- Removed unused types in the serviceaccount.ts model for frontend

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
https://github.com/grafana/grafana-enterprise/issues/2515

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: public/app/types/serviceaccount.ts [hunk 1]
```diff
@@ -48,29 +47,3 @@ export interface ServiceAccountsState {
...
-export interface ServiceaccountListAdminState {
-  serviceaccounts: ServiceAccountDTO[];
-  query: string;
-  perPage: number;
-  page: number;
-  totalPages: number;
-  showPaging: boolean;
-  filters: ServiceAccountFilter[];
-  isLoading: boolean;
-}
```
Comment (eleijonmarck) [inline public/app/types/serviceaccount.ts:76]: this can be removed for now @IevaVasiljeva

File: pkg/services/serviceaccounts/api/api.go [hunk 2]
```diff
@@ -44,6 +44,9 @@ func (api *ServiceAccountsAPI) RegisterAPIEndpoints(
 	}
 	auth := acmiddleware.Middleware(api.accesscontrol)
 	api.RouterRegister.Group("/api/serviceaccounts", func(serviceAccountsRoute routing.RouteRegister) {
+		// serviceAccountUIScope := accesscontrol.Scope("serviceaccounts", "id", accesscontrol.Parameter(":serviceAccountId"))
+		// serviceAccountsRoute.Get("/serviceAccountId", auth(middleware.ReqOrgAdmin, accesscontrol.EvalPermission(serviceaccounts.ActionRead, serviceAccountUIScope)), routing.Wrap(api.getServiceAccount))
```
Comment (eleijonmarck) [inline pkg/services/serviceaccounts/api/api.go:48]:
```suggestion
```

**Review by eleijonmarck — COMMENTED** [review]
comments and need to remove comment

**Review by donomii — APPROVED** [review]
Backend LGTM

**Pull request merged**